scripts/bin/update-checksum

```diff
@@ -57,29 +57,42 @@ for package in "${@}"; do
 
 	# Calculate new checksum in subshell.
```

Comment:
This comment could use an update.
```suggestion
	# Command substitution create a subshell, we want to do this 
	# to not effect the main scope's variables by sourcing the build script
```
Something along these lines, feel free to come up with something less wordy.